README.md

```
@@ -5,15 +5,40 @@ $ npm run dev
-    - 기능 추가 : DB에 미등록된 검색어에 대해 유효 url 탐색
+1. user_favorites 관련 코드 삭제
+    -  user_bookmarks로 대체하기 위함
+    -  notification_enabled 북마크 알림 관련 기능 제거
+    - favicon(페이지 대표 이미지) 추가 예정
+    -  사용자 설정 필요
+        ```sql
+        DROP TABLE IF EXISTS user_favorites CASCADE;
+        ```
+2. user_bookmarks 관련 기능 추가
+    - 사용자 설정 필요
+        ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)